### DISCUSSION TITLE
AI Thinker Audiokit V 2.2

### DISCUSSION THREAD

**Discussion by helste64** [pschatzmann/arduino-audio-tools#333] [Q&A] [answered]
I am using the AI Thinker Audiokit V 2.2 board and trying to upload the sketch " streams-generator_inputmixer-audiokit.ino". Unfortunately I get an error message "AudioStreams.h:277:9: error: label 'PS_RAM' defined but not used [-Werror=unused-label] PS_RAM:"
What I do wrong?

**Comment by pschatzmann** [comment, marked as answer]
In Preferences make sure that Compiler warnings is set to Default
